sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -89,8 +89,27 @@ public HttpPipeline getHttpPipeline() {
         return this.protocolLayer.getHttpPipeline();
     }
 
-    // TODO: This is a temporary implementation for sample purposes. This should be spruced up/replaced once this API is actually designed.
-    // Input is String and output is Response<String>.
+    /**
+     * Creates a digital twin.
+     *
+     * @param digitalTwinId The Id of the digital twin.
+     * @param digitalTwin The application/json digital twin to create.
+     * @return The application/json digital twin created.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<String> createDigitalTwin(String digitalTwinId, String digitalTwin)
+    {
+        return createDigitalTwinWithResponse(digitalTwinId, digitalTwin)
+            .map(DigitalTwinsResponse::getValue);
+    }
+
+    /**
+     * Creates a digital twin.
+     *
+     * @param digitalTwinId The Id of the digital twin.
+     * @param digitalTwin The application/json digital twin to create.
+     * @return A Http response containing application/json digital twin created.
```

Comment:
I don't like referring to the Response<T> as just an "Http response". How about "A {@link DigitalTwinsResponse} containing the application/json digital twin created."?

Reply:
The DigitalTwinsResponse class has more info than just the http response, after all. It also has the http request

Reply:
Yes, I agree. I like "A {@link DigitalTwinsResponse} containing the application/json digital twin created."

Reply:
yeah definitely sounds better, I'll make that change